common/src/main/java/com/wynntils/services/mapdata/providers/builtin/CategoriesProvider.java

```diff
@@ -66,11 +77,121 @@ public String getIconId() {
                 public int getPriority() {
                     return 500;
                 }
+
+                @Override
+                public MapVisibility getIconVisibility() {
+                    return DEFAULT_VISIBILITY;
+                }
+
+                @Override
+                public MapVisibility getLabelVisibility() {
+                    return DEFAULT_VISIBILITY;
+                }
+            };
+        }
+    }
+
+    private static final class WaypointCategory implements MapCategory {
+        @Override
+        public String getCategoryId() {
+            return "wynntils:personal:waypoint";
+        }
+
+        @Override
+        public String getName() {
+            return "Personal Waypoints";
+        }
+
+        @Override
+        public MapAttributes getAttributes() {
+            return new AbstractMapAttributes() {
+                @Override
+                public int getPriority() {
+                    return 500;
```

Comment:
How come waypoints are not max (1000) priority? Afaik, they render above everything else right now.